app-modules/form/src/Actions/GenerateFormKitSchema.php

```diff
@@ -58,21 +57,19 @@ public function __invoke(Submissible $submissible): array
         ];
     }
 
-    public function content(array $content, ?Collection $fields = null): array
+    public function content(array $blocks, array $content, ?Collection $fields = null): array
```

Comment:
Maybe the blocks should be passed after the content? Feels secondary to me

Reply:
I disagree because of the recursion. In my opinion the parameters at the front should be the static parameters that are used in each iteration, followed by the parameters that change each iteration, both with respect to decreasing importance.

Reply:
`$fields` is static, I think, and it's after content. But sure, lets just keep it like this for now, it's just a minor thing.

Reply:
Actually you're right. I assumed because it was optional that it was changing somewhere.

Reply:
It's set on the first iteration if it is not present, and optional params need to be at the end of the list